README.md

```diff
@@ -1,2 +1,213 @@
-# module-microsoftgraph
-Ballerina connector for Microsoft Graph API
+# Module Microsoft Graph
+
+Connects to Microsoft Graph API which acts as the gateway to data and intelligence in Microsoft 365. Microsoft Graph exposes a unified programming model which can be used to access vast amounts of data available in Office 365, Windows 10, and Enterprise Mobility & Security.
+
+The current implementation of Microsoft Graph consists of the following sub modules.
+
+**Spreadsheet Operations**
+
+The `wso2/msspreadsheets` module contains operations to perform CRUD (Create, Read, Update, and Delete) operations on [Excel workbooks](https://docs.microsoft.com/en-us/graph/api/resources/excel?view=graph-rest-1.0) stored in Microsoft OneDrive.
+
+**OneDrive Operations**
+Microsoft OneDrive is a file hosting service and synchronization service run by Microsoft. It is operated as part of Microsoft Office 365.
+The `wso2/msonedrive` module contains operations for accessing the items stored in OneDrive.
+
+## Compatibility
+|                     |    Version     |
+|:-------------------:|:--------------:|
+| Ballerina Language  | 1.2.0   |
+| Microsoftgraph REST API | v1.0          |
+
+## Getting started
+
+1.  Refer the [Getting Started](https://ballerina.io/learn/getting-started/) guide to download and install Ballerina.
```

Comment:
```suggestion
1.  Download and install Ballerina. For instructions, go to [Installing Ballerina](https://ballerina.io/v1-2/learn/installing-ballerina/).
```
We do not have a Getting Started now in the Ballerina website. This should be linked to "Installing Ballerina".

Reply:
Ok, Updated